### PR TITLE
 minor Merge and forkchoice updates release

### PR DESCRIPTION
**Pull request by djrtwo**
todo:
- [x] #2730
- [x] testgen
- [x] #2739

### CODE REVIEW

**Comment by dapplion** [comment]
@djrtwo CL clients are expected to implement https://github.com/ethereum/consensus-specs/pull/2730 at any time between now and the merge? Is there any time target or just whenever possible?

**Comment by djrtwo** [comment]
I'll put some notes in the release, but ideally, these changes are on merge testnets by roughly end of January and released on the normal release cycle at or before the Merge itself

File: configs/mainnet.yaml [hunk 1]
```diff
@@ -70,6 +70,11 @@ MIN_PER_EPOCH_CHURN_LIMIT: 4
 CHURN_LIMIT_QUOTIENT: 65536
 
 
+# Fork choice
+# ---------------------------------------------------------------
+# 25%
```
Comment (djrtwo) [inline configs/mainnet.yaml:75]:
```suggestion
# 70%
```

File: configs/minimal.yaml [hunk 2]
```diff
@@ -69,6 +69,12 @@ MIN_PER_EPOCH_CHURN_LIMIT: 4
 CHURN_LIMIT_QUOTIENT: 32
 
 
+# Fork choice
+# ---------------------------------------------------------------
+# 25%
```
Comment (djrtwo) [inline configs/minimal.yaml:74]:
```suggestion
# 70%
```

**Review by djrtwo — COMMENTED** [review]
minor typos in config yaml

File: tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_on_block.py [hunk 3]
```diff
@@ -698,15 +694,106 @@ def test_new_finalized_slot_is_justified_checkpoint_ancestor(spec, state):
...
 
     assert store.finalized_checkpoint == another_state.finalized_checkpoint
-    assert store.justified_checkpoint != another_state.current_justified_checkpoint
+    assert store.justified_checkpoint == another_state.current_justified_checkpoint
+
+    yield 'steps', test_steps
+
+
+@with_all_phases
+@spec_state_test
```
Comment (djrtwo) [inline tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_on_block.py:710]: These new tests should probably have. My test vector build is insanely slow

```python
@with_presets([MINIMAL], reason="too slow")
```

Comment (djrtwo) [reply tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_on_block.py:710]: nevermind, it's the transition tests iiuc

**Review by Mitumaru — COMMENTED** [review]
Ok

**Pull request merged**